test/unify_package_rename_suggestors/import_renamer_test.dart

```diff
@@ -0,0 +1,209 @@
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'package:over_react_codemod/src/unify_package_rename_suggestors/import_renamer.dart';
+import 'package:test/test.dart';
+
+import '../util.dart';
+
+void main() {
+  group('importRenamer', () {
+    final testSuggestor = getSuggestorTester(
+      importRenamerSuggestorBuilder(
+        oldPackageName: 'react_material_ui',
+        newPackageName: 'unify_ui',
+        oldPackageNamespace: 'mui',
+        newPackageNamespace: 'unify',
+      ),
+    );
+
+    test('does nothing when there are no imports', () async {
+      await testSuggestor(input: '');
+    });
+
+    test('does nothing for non-react_material_ui imports', () async {
+      await testSuggestor(
+        input: /*language=dart*/ '''
+            import 'package:over_react/over_react.dart';
+        ''',
+      );
+    });
+
+    // All tests strings are split by package name to work around issues with dependency_validator.
+    group('updates react_material_ui imports in a file', () {
+      test('', () async {
+        await testSuggestor(
+          input: '''
+              import 'package:over_react/over_react.dart';
+              import 'package:react_material_ui/react_material_ui.dart';
+              import 'package:react_material_ui/z_alpha_may_break_at_runtime_do_not_release_to_customers.dart';
+              import 'package:react_material_ui/components/badge.dart';
+              import 'package:react_material_ui/abc.dart';
+          
+              content() => Dom.div()();
+          ''',
+          expectedOutput: '''
+              import 'package:over_react/over_react.dart';
+              import 'package:'''
+              '''unify_ui/abc.dart';
+              import 'package:'''
+              '''unify_ui/components/badge.dart';
+              import 'package:'''
+              '''unify_ui/unify_ui.dart';
+              import 'package:'''
+              '''unify_ui/z_alpha_may_break_at_runtime_do_not_release_to_customers.dart';
+          
+              content() => Dom.div()();
+          ''',
+        );
+      });
+
+      test('in alphabetical order', () async {
+        await testSuggestor(
+          input: '''
+              import 'package:react_material_ui/react_material_ui.dart' as mui;
+              import 'package:react_material_ui/styles/styled.dart' as mui;
+          
+              content() => Dom.div()();
+          ''',
+          expectedOutput: '''
+              import 'package:'''
+              '''unify_ui/styles/styled.dart' as unify;
+              import 'package:'''
+              '''unify_ui/unify_ui.dart' as unify;
+          
+              content() => Dom.div()();
```

Comment:
#nit
```suggestion
        await testSuggestor(
          input: '''
              import 'package:react_material_ui/react_material_ui.dart' as mui;
              import 'package:react_material_ui/styles/styled.dart' as mui;
          
              content() => Dom.div()();
          ''',
          expectedOutput: '''
              import 'package:unify_ui/styles/styled.dart' as unify;
              import 'package:unify_ui/unify_ui.dart' as unify;
          
              content() => Dom.div()();
```

why is it like this? lol